src/main/java/com/tailf/jnc/Device.java

```diff
@@ -338,20 +338,37 @@ public SSHSession getSSHSession(String sessionName) {
     /**
      * This method finds the {@link DeviceUser} associated to the localUser
      * user name and SSH connects to the device, this method must be called
-     * prior to establishing any sessions (channels)
+     * prior to establishing any sessions (channels).
+     *
+     * The connection requires host key check.
```

Comment:
what if username + password is used? no host key check then?

Reply:
That's independent, but the point is that by default host key check is performed; other `connect` methods allow to skip that.